src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
 // write your code here
+const collection = document.querySelectorAll('.population');
+
+const population = [...collection]
+  .map((item) => Number(item.innerText.split(',').join('')))
```

Comment:
```suggestion
const population = collection
  .map((item) => Number(item.innerText.split(',').join('')))
```
I mean you don't need to create a copy with a spread operator

Reply:
> I mean you don't need to create a copy with a spread operator
But NodeList haven't a method like a `map`, that's why I converting this collection into array with spread operator

Reply:
> > I mean you don't need to create a copy with a spread operator
> 
> But NodeList haven't a method like a `map`, that's why I've converted this collection into array with spread operator

Ah, got it, I forgot that